emanote/src/Emanote/Model/Type.hs

```diff
@@ -243,6 +245,23 @@ modelLookupNoteByHtmlRoute r =
     . N.lookupNotesByHtmlRoute r
     . _modelNotes
 
+isNoteFeed :: Note -> Bool
+isNoteFeed note = case _noteMeta note ^? key "generate_feed" % _Bool of
+  Just True -> True
+  _ -> False
```

Comment:
Can you make this a bit static, by storing the flag in the `Note` record itself?